src/raft_net.c

```diff
@@ -77,6 +77,21 @@ static util_thread_ctx_reg_int_t
 raft_net_recovery_lreg_multi_facet_cb(enum lreg_node_cb_ops,
                                       struct lreg_value *, void *);
 
+struct raft_instance *
+raft_net_init_instance(void)
+{
+    struct raft_instance *ri = (struct raft_instance *)calloc(1,
+                                    sizeof(struct raft_instance));
+    ri->ri_store_type = RAFT_INSTANCE_STORE_POSIX_FLAT_FILE;
+    return ri;
```

Comment:
Please use `niova_calloc()`.  

The preferred indentation method here would be:
```
struct raft_instance *ri = 
    (struct raft_instance *)niova_calloc(1UL, sizeof(struct raft_instance));
```